docs/run/max-execution-time.mdx

```diff
@@ -48,10 +44,6 @@ The system calculates an appropriate job timeout value based on the input circui
 
 For example, if you specify `max_execution_time=5000` (approximately 83 minutes), but the system determines it should not take more than 5 minutes (300 seconds) to execute the job, then the job is canceled after 5 minutes.
 
-## Session maximum execution time
```

Comment:
This doesn't need to be removed. A session still has a `max_time`, and the behavior remains the same.